### ISSUE TITLE
Investigate slow builds that cause AppVeyor to quit

### ISSUE THREAD

**Issue by matozoid** [javaparser/javaparser#1513]

**Comment by matozoid** [comment]
The maven-buildtime-extension just says the obvious thing:
```
[INFO] 
[INFO] javaparser-core-testing
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [3.948s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.011s]
[INFO]   maven-resources-plugin:copy-resources (copy-resources) ... [0.136s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.002s]
[INFO]   jacoco-maven-plugin:report (jacoco-site) ................. [0.096s]
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.042s]
[INFO]   jacoco-maven-plugin:prepare-agent (jacoco-initialize) .... [0.002s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.006s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [36.009s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [0.006s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.002s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.119s]
[INFO] javaparser-symbol-solver-testing
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [1.413s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.010s]
[INFO]   maven-resources-plugin:copy-resources (copy-resources) ... [0.193s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.003s]
[INFO]   jacoco-maven-plugin:report (jacoco-site) ................. [0.001s]
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.066s]
[INFO]   jacoco-maven-plugin:prepare-agent (jacoco-initialize) .... [0.001s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.004s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [481.230s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [0.002s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.001s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.201s]
[INFO] javaparser-core-generators
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.004s]
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [0.002s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.009s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [0.004s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.022s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [0.205s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.034s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.002s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.001s]
[INFO] javaparser-symbol-solver-model
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.003s]
[INFO]   animal-sniffer-maven-plugin:check (animal-sniffer) ....... [0.877s]
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [0.001s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.009s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [0.003s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.017s]
[INFO]   maven-enforcer-plugin:enforce (enforce-versions) ......... [0.146s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [0.035s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.046s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.001s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.001s]
[INFO] javaparser-core
[INFO]   templating-maven-plugin:filter-sources (filter-src) ...... [0.008s]
[INFO]   animal-sniffer-maven-plugin:check (animal-sniffer) ....... [1.257s]
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [0.004s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.467s]
[INFO]   maven-enforcer-plugin:enforce (enforce-versions) ......... [0.194s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.091s]
[INFO]   ph-javacc-maven-plugin:javacc (javacc) ................... [0.003s]
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.035s]
[INFO]   build-helper-maven-plugin:add-source (add-source) ........ [0.001s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.015s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [0.158s]
[INFO]   bnd-maven-plugin:bnd-process (default) ................... [1.539s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [14.150s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.365s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.004s]
[INFO] javaparser-symbol-solver-core
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.008s]
[INFO]   animal-sniffer-maven-plugin:check (animal-sniffer) ....... [0.669s]
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [0.002s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.007s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [0.003s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.067s]
[INFO]   maven-enforcer-plugin:enforce (enforce-versions) ......... [0.067s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [1.471s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.113s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.001s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.002s]
[INFO] javaparser-parent
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.521s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.155s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.362s]
[INFO] javaparser-core-metamodel-generator
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.003s]
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [0.002s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.010s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [0.003s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.030s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [0.089s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.019s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.001s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.002s]
[INFO] javaparser-symbol-solver-logic
[INFO]   maven-clean-plugin:clean (default-clean) ................. [0.002s]
[INFO]   animal-sniffer-maven-plugin:check (animal-sniffer) ....... [0.718s]
[INFO]   maven-compiler-plugin:testCompile (default-testCompile) .. [0.002s]
[INFO]   maven-install-plugin:install (default-install) ........... [0.008s]
[INFO]   maven-surefire-plugin:test (default-test) ................ [0.003s]
[INFO]   maven-jar-plugin:jar (default-jar) ....................... [0.020s]
[INFO]   maven-enforcer-plugin:enforce (enforce-versions) ......... [0.084s]
[INFO]   maven-compiler-plugin:compile (default-compile) .......... [0.094s]
[INFO]   maven-source-plugin:jar (attach-sources) ................. [0.051s]
[INFO]   maven-resources-plugin:resources (default-resources) ..... [0.002s]
[INFO]   maven-resources-plugin:testResources (default-testResource [0.001s]
[INFO] ------------------------------------------------------------------------
```

**Comment by matozoid** [comment]
Check if the slow tests somehow got enabled on a normal build.

**Comment by matozoid** [comment]
Fixed in #1825
